src/wp-includes/meta.php

```diff
@@ -1241,6 +1293,13 @@ function register_meta( $object_type, $meta_key, $args, $deprecated = null ) {
 		}
 	}
 
+	if ( array_key_exists( 'default', $args ) ) {
+		if ( false === $args['single'] && ! wp_is_numeric_array( $args['default'] ) ) {
+			$args['default'] = array( $args['default'] );
+		}
+		add_filter( "default_{$object_type}_metadata", 'filter_default_metadata', 10, 5 );
```

Comment:
This will end up hooking the same function in multiple times to that hook if more than one meta is registered with defaults of the same type. I think we'd need a `has_filter` check?

Reply:
How is this different from other add_filter calls in the function?

Reply:
Because this filter is unique to the object type, the other filters are unique to the meta key.

Reply:
Change I change filter name or add `has_filter` check?

Reply:
Either works for me, a more specific filter name probably follows other patterns the closest.

Reply:
Went with `has_filter` as it is a smaller change. https://github.com/WordPress/wordpress-develop/pull/256/commits/cc6cf32546d03abd474f2ca54753cedc87903250

`default_{$object_type}_metadata` follows the pattern of 
`"add_{$meta_type}_metadata"`, `"update_{$meta_type}_metadata"` and `"delete_{$meta_type}_metadata"`